parquet/src/main/java/org/apache/iceberg/parquet/ReadConf.java

```diff
@@ -186,27 +184,14 @@ private Map<Long, Long> generateOffsetToStartPos(Schema schema) {
       return null;
     }
 
-    FileDecryptionProperties decryptionProperties =
-        (options == null) ? null : options.getDecryptionProperties();
-
-    ParquetReadOptions readOptions =
-        ParquetReadOptions.builder().withDecryption(decryptionProperties).build();
-
-    try (ParquetFileReader fileReader = newReader(file, readOptions)) {
```

Comment:
Removing the `newReader` is really the main part of this change. newReader will instantiate a new ParquetFileReader which will perform an I/O and re-read the footer which we already read.

Reply:
I see, this is somehow needed on the MoR path? Although I don't see why yet